### ISSUE TITLE
Assets are not copied to _public on save

### ISSUE THREAD

**Issue by paprikka** [scotch/angular-brunch-seed#71]
I've noticed that auto-reload.js throws 404 error every time I re-compile scripts.

At the beginning I thought there are some problems with this particular plugin but it turns out content from any other folders inside _assets_ dir is ignored.

The problem occurs on a Windows 8 machine. I have no access to my mac at the moment, so I can't check if it's a Windows-specific issue only.

![image](https://f.cloud.github.com/assets/1511643/470629/a2162bc8-b6fe-11e2-92a4-5b7353d19560.png)

In _tests_ dir only _scenario.js_ is present.

I know that there are some issues with rimraf on Windows at the moment, maybe it's related somehow?

**Comment by kylefinley** [comment]
I wonder if this is caused by the regex again: [config.coffee#L5](https://github.com/scotch/angular-brunch-seed/blob/master/config.coffee#L5)
maybe:

``` coffeescript
    assets: /^app\/assets\//
```

 should be something like:

``` coffeescript
    assets: /^app(\/|\\)assets(\/|\\)/
```

**Comment by paprikka** [comment]
Well, I've tried it before but with no success:(

Again, old versions of this repo work fine. I wonder if any other person on
win has the same problem.

## 

Rafał Pastuszak

Dnia 7 maj 2013 o godz. 21:48 Kyle Finley notifications@github.com
napisał(a):

I wonder if this is caused by the regex again:
config.coffee#L5https://github.com/scotch/angular-brunch-seed/blob/master/config.coffee#L5
maybe:

```
assets: /^app\/assets\//
```

should be something like:

```
assets: /^app(\/|\\)assets(\/|\\)/
```

—
Reply to this email directly or view it on
GitHubhttps://github.com/scotch/angular-brunch-seed/issues/71#issuecomment-17565835
.

**Comment by kylefinley** [comment]
The only other thing I can think to test is to move `/app/assets` to `/assets/` that's the default location: [configuration file](https://github.com/brunch/brunch/blob/master/docs/config.md) and delete:

```
    assets: /^app\/assets\//
```

possible related issue:
brunch/brunch/issues/548

**Comment by xixixao** [comment]
Same issue (Win7). Moving assets didn't help. Any way to solve this?

_Edit:_ For now, I have to copy the auto-reload manually...

**Comment by xixixao** [comment]
Also, auto-reload only reloads the CSS, even when I edited a partial. Is this the desired behavior?

**Comment by kylefinley** [comment]
Hey @xixixao 

Once this issue #51 is resolved manually coping `auto-reload` shouldn't be necessary. brunch-auto-reload also added [bower support](https://github.com/brunch/auto-reload-brunch/blob/master/bower.json), so you could load the script via bower if that helps.

> Also, auto-reload only reloads the CSS, even when I edited a partial. Is this the desired behavior?

Unfortunately yes, Brunch reloads on javascript changes, and auto-reload causes a reload on styles changes. Currently there is no mechanism in place to reload on partial's change.

**Comment by paulmillr** [comment]
@xixixao do you have latest auto-reload-brunch?

**Comment by xixixao** [comment]
I only installed this skeleton with all its dependencies. But a proper reload is a deal breaker for me, so back to [Mimosa](http://mimosajs.com/). Encourage you to check it out.

Also, just wandering, why is the compilation so devilishly slow? Is it because of compiling complete bootstrap? Again, Mimosa performs much better there but I haven't tried it with angular yet.

**Comment by paulmillr** [comment]
This angular skeleton is one of the biggest skeletons out there. So it is slower.

**Comment by xixixao** [comment]
Updated to latest auto-reload-brunch (the one from git) - no difference (at least for me), @paulmillr.

**Comment by kylefinley** [comment]
This should be resolved with Commit 4cffcba08874a5b101fb0ffd9b85f63576804ff6 (v0.4.0)

Please let me know if your still having problems after you upgrade.

**Comment by samora** [comment]
I am having this issue on my Windows 7 laptop. Files in `app/assets/images` are not copied to `_public` on save.
